applications/nrf_desktop/src/modules/Kconfig.hids

```diff
@@ -35,8 +15,6 @@ config DESKTOP_HID_PERIPHERAL
 	bool
 	default y
 	depends on BT_PERIPHERAL
-	depends on (DESKTOP_HID_MOUSE || DESKTOP_HID_KEYBOARD || \
```

Comment:
DESKTOP_HID_PERIPHERAL could be selected automatically based on device type (to simplify configuration)

Reply:
I prefer not - above options are not for selecting a device type but for reports. 
DESKTOP_HID_PERIPHERAL autoenables BLE related options if device is a peripheral.